### ISSUE TITLE
A11Y: Show focus state visibly

### ISSUE THREAD

**Issue by trm217** [republik/plattform#130]
> ⚠️ **contains issues that are also mentioned in #136. 
>
> Wait for #136 to be resolved and check what has already been solved.**

### Autofocus

For modals and other elements that open in a different place in the dom hierarchy the auto-focus must be set programatically.

### Make focus visible

<img width="833" alt="image" src="https://user-images.githubusercontent.com/30313631/172885778-4c5b3903-9418-4956-9002-721e63f7a509.png">

<img width="833" alt="image" src="https://user-images.githubusercontent.com/30313631/172885829-11f340b6-49f9-43e0-8c9d-0a5eb6484ed5.png">

<img width="618" alt="image" src="https://user-images.githubusercontent.com/30313631/172885888-89993dba-f0f4-4f57-adf4-6dc0820661d6.png">

[https://www.w3.org/TR/WCAG20/#understandable](https://www.w3.org/TR/WCAG20/#understandable)
[https://www.w3.org/TR/WCAG20/#consistent-behavior](https://www.w3.org/TR/WCAG20/#consistent-behavior)

## Todo

- [ ] Videos cannot be focused
<img width="730" alt="image" src="https://user-images.githubusercontent.com/30313631/172881151-2fa97086-6a3a-4fbe-b47f-75e0a2fe5f16.png">

- [ ] Unselected payment options are not focusable
<img width="730" alt="image" src="https://user-images.githubusercontent.com/30313631/172881328-ca938066-f699-4b2a-be68-aa06694a84dd.png">

 

- [ ] Auto focus Audio Player when starting a track (Befund 47  & 52)
<img width="755" alt="image" src="https://user-images.githubusercontent.com/30313631/172880618-7939f0b8-2e3b-406f-920c-e038dc915915.png">

- [ ] Focused actionbar item must be visible selected
<img width="967" alt="image" src="https://user-images.githubusercontent.com/30313631/172878886-ad3c321d-7a6e-4f68-859f-9cd04b4adcdf.png">

- [ ] Auto focus must be set when the series-overview overlay is opened in an article.
<img width="967" alt="image" src="https://user-images.githubusercontent.com/30313631/172879392-733a3c6b-53d1-4831-81bd-5fd528618428.png">

- [ ] Modals must be autofocus when opened (will require changes in Styleguide)
<img width="730" alt="image" src="https://user-images.githubusercontent.com/30313631/172882505-5d0282ce-def7-4f1c-8646-35d043db758a.png">
<img width="730" alt="image" src="https://user-images.githubusercontent.com/30313631/172882550-351edb0c-6b67-41de-baff-eb1aeb91390f.png">

- [ ] Autofocus image light-box
<img width="730" alt="image" src="https://user-images.githubusercontent.com/30313631/172883013-043746a6-3eaf-4ff5-91f5-2c56e7f297f6.png">
